src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -180,4 +181,9 @@ export class MemberListComponent implements OnChanges, OnDestroy {
       this.dataFetching.next({ groupId: this.group.id, filter: this.currentFilter, sort: this.currentSort });
     }
   }
+
+  setFilter(filter: Filter): void {
+    if (this.compositionFilter) this.compositionFilter.setFilter(filter);
```

Comment:
I suppose that is equivalent to 
```
this.compositionFilter?.setFilter(filter);
```
which is a bit shorter/cleaner

Reply:
fixed